server/helpers/artistsFuncs.js

```diff
@@ -0,0 +1,101 @@
+const { setMaxListeners } = require('superagent');
+const artists = require('../data/artists');
+
+/**
+ * Gets a random number from a given interval.
+ * 
+ * @param {The minimum value to define the lower boundary of the interval} min 
+ * @param {The maximum value to define the upper boundary of the interval} max 
+ * @returns A random int from the given interval.
+ */
+ function randomIntFromInterval(min, max) {
+    return Math.floor(Math.random() * (max - min + 1) + min);
+}
+
+
+/**
+ * Finds an artist using the given name.
+ * 
+ * @param {The name (string) of the artist to retrieve} name 
+ * @returns The artist object that matches the name.
+ */
+function findArtistByName(name) {
+    let result = null;
+
+    artists.forEach(artist => {
+        if(artist.name.toLowerCase() === name.toLowerCase()) result = artist;
+    })
+    return result;
+}
+
+/**
+ * Search for the artists whose names match the given query string
+ * 
+ * @param {The input from the user to use to search the artists} queryStr 
+ * @returns The list of artists which match the given string. Null if no results were found.
+ */
+function searchArtist(queryStr) {
```

Comment:
the approach with the search function is interesting, it seems like you have different responses coded out depending on a case by case basis based on the terms included in the queryStr